src/pages/journal/components/test-slot/test-slot.scss

```diff
@@ -1,2 +1,13 @@
 test-slot {
+  ion-row {
```

Comment:
A custom class would be good here, so that it doesn't effect the ion-rows in sub components

Reply:
Done.